activemodel/lib/active_model/validations.rb

```diff
@@ -466,6 +480,14 @@ def initialize(model)
       super(I18n.t(:"#{@model.class.i18n_scope}.errors.messages.model_invalid", errors: errors, default: :"errors.messages.model_invalid"))
     end
   end
+
+  class ValidationContext # :nodoc:
+    attr_accessor :context
+
+    def initialize(context = nil)
+      @context = context
+    end
```

Comment:
Do we even call this method with a context? If not, do we need the intialize?

Reply:
Hey @rafaelfranca! Good point. Ill remove it